dev-libs/botan/botan-2.0.1.recipe

```diff
@@ -0,0 +1,71 @@
+SUMMARY="A cryptography library written in C++"
+DESCRIPTION="This is the best option for crypthography in C++"
+HOMEPAGE="https://botan.randombit.net/"
+COPYRIGHT="2000-2017 botan"
+LICENSE="BSD (2-clause)"
+REVISION="1"
+SOURCE_URI="https://github.com/randombit/botan/archive/$portVersion.tar.gz"
+CHECKSUM_SHA256="f4c02bc4247eb78ac004ce10374992dd488f0d1ba21e712874cd77481332f731"
+SOURCE_DIR="botan-$portVersion"
+
+ARCHITECTURES="!x86_gcc2 !x86 !x86_64"
+SECONDARY_ARCHITECTURES="!x86"
+
+PROVIDES="
+	botan$secondaryArchSuffix = $portVersion
+	lib:libbotan_2$secondaryArchSuffix = 0.0.1 compat >= 0
+	cmd:botan$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	botan${secondaryArchSuffix}_devel = $portVersion
+	devel:libbotan_2$secondaryArchSuffix = 0.0.1 compat >= 0
+	"
+REQUIRES_devel="
+	haiku${secondaryArchSuffix}_devel
+	botan$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libbz2$secondaryArchSuffix
+	devel:libcrypto$secondaryArchSuffix
+	devel:liblzma$secondaryArchSuffix
+	devel:libsqlite3$secondaryArchSuffix
+	devel:libssl$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	cmd:g++$secondaryArchSuffix
+	cmd:make
+	cmd:python
+	"
+PATCH()
+{
+	#	sed -i -e "/^all\>/ s/-pthread //" src/build-data/cc/gcc.txt
+	sed -i -e "/^all\>/ s/-pthread -fstack-protector/-fno-stack-protector/" \
+		src/build-data/cc/gcc.txt
+}
+
+BUILD()
+{
+	runConfigure --omit-buildspec ./configure.py \
+		--with-bzip2 --with-lzma --with-zlib \
+		--with-openssl --with-sqlite3 \
+		--destdir=/
```

Comment:
Thanks @pulkomandy for the suggestion.
I tried with a "libtoolize --force --copy --install" a few days ago, but it did not help.
I'll push it here, with the fixes I had requested in my [last review](#pullrequestreview-17802804), and maybe we can merge this PR even if the recipe does not yet build the shared lib.